metricbeat/module/ceph/osdpoolstats/_meta/data.json

```diff
@@ -0,0 +1,19 @@
+{
+    "@timestamp":"2016-05-23T08:05:34.853Z",
+    "beat":{
+        "hostname":"beathost",
+        "name":"beathost"
+    },
+    "metricset":{
+        "host":"localhost",
+        "module":"mysql",
```

Comment:
you can generate this files by running `make generate-json` if you have created the `integration` test files which are responsible to create the data (see other metricsets). This requires a working docker environment.